runtime/isp_utils.py

```diff
@@ -61,14 +61,15 @@ def setupLogger(level, colors=True):
     return logger
 
 
-def generateTagInfo(exe_path, run_dir, policy_dir, soc_cfg=None):
+def generateTagInfo(exe_path, run_dir, policy_dir, arch="rv32", soc_cfg=None):
```

Comment:
I don't see a need for a default parameter when the option itself has a default value

Reply:
Would following the pattern that `soc_cfg` uses be better?

Reply:
You can just omit the default parameter, e.g. `policy_dir, arch, soc_cfg=None`

Reply:
In attempting to run on the FPGA it seems like the parameter order might be important to maintain.  We'll look at it though.

Reply:
It breaks `isp_vcs.py` and `isp_vcu118.py` in `hope-gfe` because their `soc_cfg` argument would get passed to arch instead. Moving arch to the end and adding the default of none means nothing is broken.

Reply:
Ah, gotcha. This isn't too big of an issue if that's the case.